packages/server-wallet/e2e-testing/scripts/start-load-node.ts

```diff
@@ -129,6 +147,8 @@ async function setupNode(): Promise<WalletLoadNode> {
   for (const {messagePort} of peers) {
     await serverNode.registerMessagePeer(messagePort);
   }
+  const dropRate = dropRatePercentage === 0 ? 0 : dropRatePercentage / 100;
```

Comment:
Why do we need to check if the dropRatePercentage is `0`?

Reply:
Oops, we definitely don't need to do that.

Reply:
Fixed in 4d37d37